app/debrief_gui.py

```diff
@@ -31,9 +30,11 @@ def __init__(self, services):
         self.file_svc = services.get('file_svc')
         self.log = logging.getLogger('debrief_gui')
 
-        # suppress Python Image Library debug logs
+        # suppress Python Image Library and svglib debug logs
         pil = logging.getLogger('PIL')
         pil.setLevel(logging.INFO)
+        svglib = logging.getLogger('svglib')
+        svglib.setLevel(logging.INFO)
```

Comment:
if you are doing it twice, make it a function

Reply:
oops, should be fixed in the most recent commit.